gunicorn/tests/test_integration_e2e.py

```diff
@@ -23,8 +19,11 @@ def _assert_metrics(aggregator):
 
 
 def test_gunicorn(aggregator, setup_gunicorn):
-    check = GUnicornCheck(CHECK_NAME, {}, {})
-    check.check(INSTANCE)
+    instance = INSTANCE.copy()
+    instance['gunicorn_bin_path'] = setup_gunicorn['gunicorn_bin_path']
+
+    check = GUnicornCheck(CHECK_NAME, {}, [instance])
+    check.check(instance)
```

Comment:
Would be great to test with e2e tests? That would require to set up `gunicorn_bin_path` to `docker exec ...`

Reply:
👍 

Implemented with integration test, see code update.

At the moment, can't test in "e2e" with real agent since we can't retrieve metadata for the moment from the agent.

Reply:
🤦‍♂ You're right, forgot about that